### DISCUSSION TITLE
Help with trace

### DISCUSSION THREAD

**Discussion by Hostely** [netbox-community/netbox#11263] [Q&A]
I can't trace my connections in Netbox, i want to see the trace beetween a device rear port to another device rear port. But we have cross-device between them. When i click on the button trace, nothing appear.  

Do you know why ? Or the trace it's just for the interfaces.

Thank you for the help

**Comment by dreng** [comment]
Hi,

You can trace ports but only if at least one of the ports is connected to an interface. 

However, if you use the NetBox-Topology-Views plugin, you can display the connections graphically, even if no interface is connected.
